libc-top-half/musl/src/thread/pthread_atfork.c

```diff
@@ -47,3 +48,10 @@ int pthread_atfork(void (*prepare)(void), void (*parent)(void), void (*child)(vo
 	UNLOCK(lock);
 	return 0;
 }
+#else
+#include "pthread_impl.h"
+int pthread_atfork(void (*prepare)(void), void (*parent)(void), void (*child)(void))
+{
+	return ENOMEM;
+}
+#endif
```

Comment:
When the entire file is different like this perhaps it makes sense to instead use a new file in the stubs directory?  Or in the bottom half?